scripts/irods/test/test_itree.py

```diff
@@ -66,6 +66,30 @@ def test_pattern_and_ignore_options_together__6791_6806(self):
         self.admin.assert_icommand(["itree",  "--regex-extended", "--ignore-regex", ".*[3-9]", "--pattern-regex", "ba(r|t)[[:digit:]]"],
                                     "STDOUT_MULTILINE", to_catch, use_regex = True)
 
+    def _verify_absolute_object_path(self, logical_path, must_exist):
+        self.assertEqual(logical_path[:1], '/', 'logical_path must start with "/"')
+        if must_exist:
+            self.admin.assert_icommand(['ils', logical_path], 'STDOUT_SINGLELINE', logical_path, desired_rc = 0)
```

Comment:
Consider using `self.<non_admin>` instead of `self.admin` for all tests.

We've missed things in the past by only running as the rodsadmin user. Doing things as a rodsuser helps us catch problems earlier.

Reply:
Should I change the tests for this issue to use a non-admin, but leave all of the existing tests as is ?  (None of the other `test_itree` test-cases run with anything but self.admin )

Reply:
@korydraughn - I ask because altering all of the tests in the itree suite may not be  an appropriate thing to drop into this pull request...?

Reply:
No, I was referring to only the tests you added.

Reply:
Done